diesel_compile_tests/tests/fail/as_changeset_deprecated_column_name.rs

```diff
@@ -16,4 +16,6 @@ struct UserForm {
     name: String,
 }
 
-fn main() {}
+fn main() {
+    compile_error!("warnings");
```

Comment:
Could we add a comment here that this is required to actually check for the warnings? Maybe even link the upstream `trybuild` issue?